src/main/java/ch/wisv/areafiftylan/controller/CurrentUserRestController.java

```diff
@@ -126,4 +130,11 @@ public Order getOpenOrder(Authentication auth) {
         UserDetails currentUser = (UserDetails) auth.getPrincipal();
         return orderService.getOpenOrder(currentUser.getUsername());
     }
+
+    @RequestMapping(value = "/seat", method = RequestMethod.GET)
+    public Seat getCurrentUserSeat(Authentication auth) {
```

Comment:
We decided that seats are linked to tickets. How would this work if 1 user has multiple tickets?